src/components/theme/contrast-colors.js

```diff
@@ -0,0 +1,82 @@
+/* eslint-disable import/prefer-default-export */
+// @flow
+import Color from 'color';
+import type { ColorType } from 'color';
+import * as vars from '../../styles/variables';
+
+// When converting from rgb/hsl to hex there is potential for
+// flattening of the color space, so we add an offset factor to account for it.
+const OFFSET_FACTOR = 0.05;
+export const MIN_CONTRAST = 4.5;
+
+// The yiq coefficients in the color library are incorrect
```

Comment:
What is `yiq`?

Reply:
This is the way we're checking the luminance of a particular colorKey to determine if it's a light or dark color.